modules/building/pages/creating.adoc

```diff
@@ -93,59 +99,71 @@ $ kubectl apply -f Application.yaml -f Component.yaml -f ImageRepository.yaml
 
-In quay.io web UI, go to your organization and click on *Create New Repository*, enter repository name
-and visibility and click on *Create Public/Private Repository* button.
-eg. `quay.io/test_organization/own-repository`
+. In quay.io web UI, go to your organization and click on *Create New Repository*, enter the repository name
+and visibility, and click on *Create Public/Private Repository*.For example, enter `quay.io/ test_organization/own-repository`
 
-Create robot account in your organization and grant write permission to your repository,
-you can also create another robot account only with read permissions.
+. Create a robot account in your organization and grant write permission to your repository. You can also create another robot account with only read permissions.
 
-In quay.io web UI, go to your organization, click on *Robot Accounts* in the left menu,
-then click on *Create Robot Account*, enter name of robot account and click on *Create robot account* button,
-in the list of repos find your repository and set *write* permission for it and then click on *Add permissions* button.
+. In the quay.io web UI, go to your organization, click on *Robot Accounts* in the left menu,
+then click on *Create Robot Account*. 
+. Enter the name of the robot account and click on the *Create robot account* button. 
+. In the list of repos, find your repository and set *write* permission for it, and then click on the *Add permissions* button.
++
 
 When you click on the newly created robot account on *Robot Accounts* page, you can get details of the account.
-In the openened popup for the account first you will see credentials *Username* and *Token* (we will use them later),
-when you click on *Kubernetes Secret* you can download secret yml file (we will use it later).
 
-Now we will have to create secret with repository credentials.
+. In the opened popup for the account, first you will see the credentials *Username* and *Token* (take note of these as they will be used later).
+
+. Click on *Kubernetes Secret* to download a secret yaml file (it will be used later in this procedure).
+
+Next, you have to create secret with repository credentials.
+
+=== Creating a Secret via {ProductName} UI
+
+.*Procedure*
+
+. xref:building:creating-secrets.adoc#example-of-creating-a-quay-io-secret[Create a secret] with the credentials above.
++
 
-=== Creating Secret via {ProductName} UI
-You can now xref:building:creating-secrets.adoc#example-of-creating-a-quay-io-secret[create a secret]
-with the credentials above.
 Make sure to use the most narrowly valid scope for the `registry server address` field
 when adding the secret.
++
+
+When adding the secret (image registry) with the UI, you have to choose components to which component-specific
+service accounts `build-pipeline-$COMPONENT_NAME` the secret will be linked to. You do this so that the secret can be leveraged by builds.
```

Comment:
@dirgim , do we have documentation for how to link the secret to an appropriate service account used for the integration tests too? Is this done automagically with the UI? Do we need to describe the process if done manually?